src/software/embedded/services/motor.h

```diff
@@ -113,31 +113,96 @@ class MotorService
      */
     void setup();
 
+    /**
+     * Disables motors not in enabled_motors
```

Comment:
I would try to avoid referring to private variables (`enabled_motors`) in public method docs

Reply:
agreed, probably should be private